### PR TITLE
AA - Edit create/edit page with multiple categories

### PR DESCRIPTION
**Pull request by AntonovAtanas**
- Edit create/edit page with multiple categories
- Add new service updateAuthData
- Fixed toaster time

### CODE REVIEW

File: src/app/events/event-detail/event-detail.page.html [hunk 1]
```diff
@@ -22,7 +22,9 @@
     </div>
 
     <div class="category-city-wrapper">
-      <span class="badge">{{event.category}}</span>
+      <ng-container *ngFor="let category of event.category">
```
Comment (Getsov) [inline src/app/events/event-detail/event-detail.page.html]: event.categories is the right one, I guess?

File: src/app/home/slide-card/slide-card.component.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export class SlideCardComponent implements OnInit {
       phone: '',
       email: '',
     },
-    category: '',
+    category: [],
```
Comment (Getsov) [inline src/app/home/slide-card/slide-card.component.ts]: categories?

File: src/app/events/event-detail/event-detail.page.ts [hunk 3]
```diff
@@ -41,7 +41,7 @@ export class EventDetailPage implements OnInit {
       phone: '',
       email: '',
     },
-    category: '',
+    category: [],
```
Comment (Getsov) [inline src/app/events/event-detail/event-detail.page.ts]: categories?

File: src/shared/interfaces/Event.ts [hunk 4]
```diff
@@ -16,7 +16,7 @@ export interface Event {
     phone: string;
     email: string;
   };
-  category: string;
+  category: string[];
```
Comment (Getsov) [inline src/shared/interfaces/Event.ts]: categories?

Comment (AntonovAtanas) [reply src/shared/interfaces/Event.ts]: Fixed in plural.

**Review by Getsov — APPROVED** [review]

**Pull request merged**